desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,33 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
+- Add Docker Desktop version to 'docker version' output.
+
+### Upgrades
+
+- [Amazon ECR Credential Helper v0.6.0](https://github.com/awslabs/amazon-ecr-credential-helper/releases/tag/v0.6.0){: target="blank" rel="noopener" class=""}
+
+### Bug fixes and minor changes
+
+- Fixed VM startup hang when using `osxfs` and no host directories are shared with the VM.
+- Increased filesystem watch ("inotify") limits by setting `fs.inotify.max_user_watches=1048576` and 'fs.inotify.max_user_instances=8192' in Linux.
+- Fixed an issue related to compose app started with version 2, but the dashboard only deals with version 1.
```

Comment:
```suggestion
- Fixed an issue that didn't allow users to stop a Docker Compose application using Docker Dashboard if the application was started in a different version of Docker Compose. For example, if the user started a Docker Compose application in V1 and then switched to Docker Compose V2, attempts to stop the Docker Compose application would fail. 
```

Reply:
@mat007 WDYT?